aries-site/src/examples/templates/table-customization/components/TableCustomizationExample.js

```diff
@@ -100,38 +100,42 @@ export const TableCustomizationExample = () => {
       background="background"
     >
       <FiltersProvider>
-        <Header pad={{ top: 'medium' }}>
-          <Box gap="xsmall" fill="horizontal">
-            <Heading level={2} margin="none">
-              Users
-            </Heading>
-            <Box direction="row" justify="between">
-              <FilterControls
-                data={allData}
-                filters={filtersConfig}
-                searchFilter={{ placeholder: 'Search users...' }}
-              />
-              <Box direction="row" height="fit-content">
-                <DropButton
-                  icon={<Edit />}
-                  dropAlign={{ top: 'bottom', right: 'right' }}
-                  onClose={() => setOpen(false)}
-                  onOpen={() => setOpen(true)}
-                  dropContent={
-                    <ColumnSettings
-                      columns={COLUMNS}
-                      visibleColumns={visibleColumns}
-                      setVisibleColumns={setVisibleColumns}
-                      open={open}
-                    />
-                  }
-                />
-                <Menu label="Actions" items={[]} />
+        <Box gap="medium">
+          <Header pad={{ top: 'medium' }}>
+            <Box gap="xsmall" fill="horizontal">
+              <Heading level={2} margin="none">
+                Users
+              </Heading>
+              <Box direction="row" justify="between" align="start">
+                <Box direction="row" align="start" gap="small">
+                  <FilterControls
+                    data={allData}
+                    filters={filtersConfig}
+                    searchFilter={{ placeholder: 'Search users...' }}
+                  />
+                  <DropButton
+                    icon={<Configure />}
+                    kind="toolbar"
+                    dropAlign={{ top: 'bottom', right: 'right' }}
+                    onClose={() => setOpen(false)}
+                    onOpen={() => setOpen(true)}
+                    dropContent={
+                      <ColumnSettings
+                        columns={COLUMNS}
+                        visibleColumns={visibleColumns}
+                        setVisibleColumns={setVisibleColumns}
+                        open={open}
+                      />
+                    }
+                    tip="Configure columns"
```

Comment:
```suggestion
                    tip="configure columns"
```

Reply:
to align more with "open filters"

Reply:
Wondering if both should be capitalized? "Open filters" and "Configure columns"?

Reply:
https://github.com/grommet/grommet/blob/master/src/js/languages/default.json has mostly lowercase. I do think it would be good to be consistent either way.

Reply:
Some of those are only read allowed by screen reader but not visible in the UI, where as these tips are visible when the user hovers over the button. It feels that since the tooltip in the UI is a message, it would make sense to sentence case them.

Reply:
works for me